src/main/java/com/mservicetech/openapi/validation/OpenApiValidator.java

```diff
@@ -44,6 +44,7 @@ public class OpenApiValidator {
     final String STATUS_INVALID_REQUEST_PATH = "ERR10007";
     final String STATUS_METHOD_NOT_ALLOWED = "ERR10008";
     final String STATUS_CONTENT_TYPE_MISMATCH = "ERR10015";
+    final String STATUS_CONTENT_TYPE_NOT_SUPPORTED = "ERR11108";
```

Comment:
needs https://github.com/networknt/light-4j/pull/2299

Reply:
I will release a snapshot version this evening. Thanks.